### DISCUSSION TITLE
How to obtain the FairnessDashboard port?

### DISCUSSION THREAD

**Discussion by DevLugo** [fairlearn/fairlearn#725] [Q&A] [answered]
Hi guys.

I already know that when you create an instance of FairnessDashboard this prints a message with the port.

`web_1    | Fairness started at http://localhost:5002`

But I want to obtain the port and store it on a variable.

Any way to obtain the port of a FairnessDashboard instance?

Thanks!

**Comment by romanlutz** [comment, marked as answer]
Hi @DevLugo !
Sounds like you're using `raiwidgets` since `FairnessDashboard` only exists there, while `fairlearn.widget` only has a `FairlearnDashboard` (which is deprecated in favor of the `raiwidgets` `FairnessDashboard`). Please open an issue in the [corresponding repo](https://github.com/microsoft/responsible-ai-widgets) and we can help you there.

While you're here: the following should solve your problem
```
dashboard = FairnessDashboard(...)
print(dashboard._service.port)
```
If that's not sufficient please open an issue in the `raiwidgets` repo. Thanks!
